src/Mapbender/PrintBundle/Resources/public/element/printclient.job-list.js

```diff
@@ -159,12 +159,12 @@ $.widget("mapbender.mbPrintClientJobList", {
                 .attr('target', '_blank')
                 .attr('title', Mapbender.trans('mb.print.printclient.joblist.open'))
             ;
-            $icon = $('<i/>').addClass('fa fa-file-pdf-o');
+            $icon = $('<i/>').addClass('fa far fa-file-pdf fa-lg');
```

Comment:
fa is superflous after adding "far", can be deleted

Reply:
done